airgun/views/computeresource.py

```diff
@@ -83,10 +83,10 @@ class GCEProviderForm(View):
         google_project_id = TextInput(id='compute_resource_project')
         client_email = TextInput(id='compute_resource_email')
         certificate_path = TextInput(id='compute_resource_key_path')
-        load_zones = Text("//*[contains(@id,'test_connection_button')]")
 
         @View.nested
         class zone(View):
+            load_zones = Text("//*[contains(@id,'test_connection_button')]")
```

Comment:
Please don't use '*' while adding locator, it is not recommended slows the locating strategy.  Please use direct tags or do you have any issue while adding tag name?